documentation/modules/con-securing-kafka-authentication.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// assembly-securing-kafka-brokers.adoc
+
+[id='con-securing-kafka-authentication-{context}']
+= Listener authentication
+
+You can create `plain` (without encryption) or `tls` listener types.
+For clients outside the Kubernetes cluster, you create `external` listeners and specify a connection mechanism, which can be `nodeport`, `loadbalancer`, `ingress` or `route` (on OpenShift).
+
+Supported authentication options:
+
+. Mutual TLS authentication (only on the listeners with TLS encryption)
+. SCRAM-SHA authentication
+. xref:assembly-oauth-authentication_str[OAuth 2.0 token based authentication]
+
+.Kafka listener authentication options
+image::listener-config-options.png[options for listener authentication configuration]
+
+The listener `authentication` property is used to specify an authentication mechanism specific to that listener.
+
+The listener authentication mechanism is defined by the `type` field.
+If no `authentication` property is specified then the listener does not authenticate clients which connect through that listener.
+The listener will accept all connections without authentication.
+
+Authentication must be configured when using the User Operator to manage `KafkaUsers`.
+
+The following example shows:
+
+* A `plain` listener configured for SCRAM-SHA authentication
+* A `tls` listener with mutual TLS authentication
+* An `external` listener with mutual TLS authentication
+
+.An example showing listener authentication configuration
+[source,yaml,subs="attributes+"]
+----
+# ...
+listeners:
+  plain:
+    authentication:
+      type: scram-sha-512
+  tls:
+    authentication:
+      type: tls
+  external:
+    type: loadbalancer
+    tls: true
+    authentication:
+      type: tls
+# ...
+----
+
+[id='con-mutual-tls-authentication-{context}']
+== Mutual TLS authentication
+
+Mutual TLS authentication is always used for the communication between Kafka brokers and ZooKeeper pods.
+
+For mutual authentication or two-way authentication, both the server and the client present certificates.
+Strimzi can configure Kafka to use TLS (Transport Layer Security) to provide encrypted communication between Kafka brokers and clients either with or without mutual authentication. When you configure mutual authentication, the broker authenticates the client and the client authenticates the broker.
+
+NOTE: TLS authentication is more commonly one-way, with one party authenticating the identity of another. For example, when HTTPS is used between a web browser and a web server, the server obtains proof of the identity of the browser.
+
+.When to use mutual TLS authentication for clients
```

Comment:
I think we should either make it a heading or remove it completely. In the current state, it is not clickable and cannot be linked but it also doesn't add any value because the text is saying basically the same as the sentence below.

Reply:
removed